sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseService.java

```diff
@@ -251,40 +251,29 @@ public List<Case> findBy(CaseCriteria caseCriteria, boolean ignoreUserFilter) {
 		return em.createQuery(cq).getResultList();
 	}
 
-	public List<Case> getAllActiveCasesAfter(Date date, boolean includeExtendedChangeDateFilters, Integer batchSize, String lastSynchronizedUuid) {
-
-		CriteriaBuilder cb = em.getCriteriaBuilder();
-		CriteriaQuery<Case> cq = cb.createQuery(getElementClass());
-		Root<Case> from = cq.from(getElementClass());
-		from.fetch(Case.SYMPTOMS);
-		from.fetch(Case.THERAPY);
-		from.fetch(Case.HEALTH_CONDITIONS);
-		from.fetch(Case.HOSPITALIZATION);
-		from.fetch(Case.EPI_DATA);
-		from.fetch(Case.PORT_HEALTH_INFO);
-		from.fetch(Case.MATERNAL_HISTORY);
+	@Override
+	@SuppressWarnings("rawtypes")
+	protected Predicate createRelevantDataFilter(CriteriaBuilder cb, CriteriaQuery cq, From<?, Case> from) {
 
 		Predicate filter = createActiveCasesFilter(cb, from);
```

Comment:
Not introduced in this commit:
The methods `createActiveCasesFilter(CriteriaBuilder cb, From<?, Case> root)` and `createActiveCasesFilter(CriteriaBuilder cb, Join<?, Case> join)` are redundant (`Join` extends `From`, code is identical, see `CaseService` l.955 ff)

Reply:
True, it's breaking (as almost every ticket changes the Facade interfaces), but I don't see a point to document this. It's not used outside our stack (as far as we know), the REST endpoints are stable and alternative Facade methods exist.

Reply:
Good finding, removed the duplicate method